src/internal/types.ts

```diff
@@ -1,4 +1,4 @@
-export type DeviceType = 'Handset' | 'Tablet' | 'Tv' | 'Desktop' | 'unknown';
+export type DeviceType = 'Handset' | 'Tablet' | 'Tv' | 'Desktop' | 'GamingConsole' | 'unknown';
```

Comment:
Adding that type means an update to README as well for return type though. Ugh - shame I don't have typedoc set up on this project. Another day - simple update

Reply:
Updated readme